src/shared/components/challenge-detail/Checkpoints/index.jsx

```diff
@@ -1,19 +1,15 @@
 import React from 'react';
 import PT from 'prop-types';
 
-import jquery from 'jquery';
-
 import './styles.scss';
 
-function decodeEscaped(escaped) {
-  return jquery('<textarea />').html(escaped).text();
-}
-
 const Checkpoints = (props) => {
   const {
     checkpointResults,
     generalFeedback,
   } = props.checkpoints;
+
+  /* eslint-disable react/no-danger, no-undef */
```

Comment:
1. Please, avoid disabling eslint rules for entire file. Either disable them for specific lines, adding `// eslint-disable-line react/no-danger` to the end of that line; or, if this results in a way too long line (> 80 columns), disable it right before the line(s) where it is used, and re-enable right after.

2. Don't disable `no-undef` rule, just add to the beginning of the file (under header comment, if any) `/* global document */`